Prace_domowe/Praca_domowa3/Grupa1/Dawid Janus/Praca domowa nr 3.ipynb

```diff
@@ -0,0 +1,514 @@
+{
```

Comment:
<p>dobrze, że obudowano to w funkcję.</p><p>Z uwagi na niezbalansowanie klas, accuracy nie jest najlepszą miarą.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/174/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>